src/status_im/contexts/shell/share/wallet/component_spec.cljs

```diff
@@ -37,12 +38,7 @@
                  (h/is-truthy (h/query-by-text "0x707f635951193ddafbb40971a0fcaab8a6415160"))
                  (h/is-falsy (h/query-by-text "eth:"))))))
 
-  ;; NOTE: Fails with error below possibly due to Infura outage:
-  ;;    FAIL  ./status_im.contexts.shell.share.wallet.component_spec.js
-  ;;  ● share wallet addresses › should display the multichain account
-  ;;
-  ;;   No protocol method IDeref.-deref defined for type undefined
-  (h/test-skip "should display the multichain account"
+  (h/test "should display the multichain account"
```

Comment:
This error isn't due to Infura, the error is because we are trying to perform a `deref` (`@`) on a `nil` value.

As you may know, we use `rf/sub` to subscribe, it's a wrapper that performs a `deref` on a call to `(rf/subscribe [...])`, so when we have no subscription we are calling `(deref nil)`.

cc: @J-Son89 @siddarthkay